### PR TITLE
Variant consolidation for answer distribution

### PR DESCRIPTION
**Pull request by dylanrhodes**
This exposes the answer distribution API's new consolidate parameter to the client.  A description of the parameter's functionality can be found in that PR <a href="https://github.com/edx/edx-analytics-data-api/pull/52">here</a>.  Feedback encouraged! @jbau @dcadams @sarina @clintonb

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @dylanrhodes! I've created [OSPR-343](https://openedx.atlassian.net/browse/OSPR-343) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams. 

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ('this must be merged by XX date', and why that is)
- partner information ('this is a course on edx.org')
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the Github pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by sarina** [comment]
Hey @dylanrhodes - help me a bit here with context, since I'm unfamiliar with Analytics work. Is this a new feature? Have you talked with the edX product team, and if not, do they need to be involved?

**Comment by dylanrhodes** [comment]
Hey @sarina this PR is just to expose the new parameter from the PR open against the data API repository (linked in the top comment).  A good explanation and discussion of what that parameter does for the API is linked there.

File: analyticsclient/module.py [hunk 1]
```diff
@@ -18,14 +20,20 @@ def __init__(self, client, course_id, module_id):
...
         """
         Get answer distribution data for a module.
 
         Arguments:
             data_format (str): Format in which to return data (default is JSON)
+            consolidate (bool): Consolidate erroneously random answers (default is False)
         """
-        path = 'problems/{0}/answer_distribution/'.format(self.module_id)
+        params = {
+            'consolidate': consolidate,
```
Comment (clintonb) [inline analyticsclient/module.py]: consolidate_variants

**Comment by dylanrhodes** [comment]
I am going to close this pull request without merging, since @shnayder and @jbau have opted to not parameterize the consolidate logic and instead make it mandatory for all problem/answer distribution API calls.  For more information, see <a href="https://github.com/edx/edx-analytics-data-api/pull/52">here</a>.

**Comment by sarina** [comment]
@dylanrhodes cool, thanks for proactively closing. Please delete your branch, as well, if it is no longer needed.

**Pull request closed without merging**